clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -944,7 +949,7 @@ int handleTimeouts(Collection<Call> calls, String msg) {
                 Call call = iter.next();
                 int remainingMs = calcTimeoutMsRemainingAsInt(now, call.deadlineMs);
                 if (remainingMs < 0) {
-                    call.fail(now, new TimeoutException(msg + " Call: " + call.callName));
+                    call.fail(now, new TimeoutException(msg + " Call: " + call.callName,call.lastThrowable));
```

Comment:
I think it's better to clarify it as "the last error causing retry is..", also nit: space after `,`.